src/model/comment.ml

```diff
@@ -40,7 +40,9 @@ type module_reference = {
 
 type nestable_block_element =
   [ `Paragraph of paragraph
-  | `Code_block of string with_location option * string with_location
+  | `Code_block of
+    string with_location option * string with_location
```

Comment:
The meaning of the first argument changed (from the whole metadata to just the language tag) but it wasn't used anywhere.

Reply:
I suggest turning it into a (new) record type.

Reply:
Good idea :)